### DISCUSSION TITLE
Template or Framework including User "sharing" access (e.g. View, Collaborate...) with their data???

### DISCUSSION THREAD

**Discussion by callagga** [vercel/next.js#70369] [Help]
### Summary

Is there a popular out of the box Next JS template/github project/larger framework that covers off having built in support for a logged in user of the Web Application (built using Next JS) to be able to "share" different levels of access to their data?    So this would imply an authentication approach (ideally with multiple option), authorisation levels (e.g. Admin, Viewer, Collaborator) and then "sharing" approach.    (i.e. to jump start development of a website that needs this and/or learning from it)

**Comment by Pyr33x** [comment]
I'm not sure if there would be a next.js template with all of those features that you determined, but creating a next.js app using [t3](https://create.t3.gg/) might be a good start for you.
